src/DynamoCoreWpf/ViewModels/PackageManager/PublishPackageViewModel.cs

```diff
@@ -793,6 +793,8 @@ private void ClearAllEntries()
             this.Assemblies = new List<PackageAssembly>();
             this.SelectedHosts = new List<String>();
             this.SelectedHostsString = string.Empty;
+            this.copyrightHolder = "";
+            this.copyrightYear = "";
```

Comment:
Let's using string.empty

Reply:
We have used "" in all the variables above this too, do you want me to update those as well?

Reply:
Done

Reply:
@zeusongit That would be even better, thanks!